documentation/user_docs/docs/manual/Cutting_data_of_interest_from_SQW_files_and_objects.rst

```diff
@@ -159,47 +163,34 @@ Each can independently have one of four different forms below.
    to ensure your cut is what you expect.
 
 
-* ``[]``
-
-  An empty binning range will use the source binning axes in that dimension.
-
-* ``[step]``
-
-  A single (scalar) number defines a plot axis.
-
-  The bin width will be equal to the number you specify. The lower and upper
-  limits will be from the source binning axes in that dimension.
-
-.. note::
-
-   A value of ``[0]`` is equivalent to ``[step]`` using bin size of the source
-   coordinate system.
-
-.. warning::
+- ``[]`` Automatic binning calculations. 
+  Empty brackets indicate that the cut algorithm should identify binning ranges itself.
+  The step size of the target binning is taken equal to the step size of the binning 
+  currently present in the source image in the direction with the same 
+  number as the number of binning argument ``[]``. In more details auto-binning algorithm is described
+  :ref:`below <bin_ranges_calculations>`.
 
-   When changing projections e.g. ``line_proj`` to ``sphere_proj``, using either
-   ``[]`` or ``[step]`` see, `changing projections`_ below.
 
```

Comment:
the same algorithm used regarless of source and target projection. Correct description recovered